app/assets/stylesheets/application.scss

```diff
@@ -1,20 +1,17 @@
-$govuk-assets-path: "/manage-transit-movements/unloading/assets/lib/govuk-frontend/govuk/assets/";
-$hmrc-assets-path: "/manage-transit-movements/unloading/assets/lib/hmrc-frontend/hmrc/assets/";
-
-@import "lib/govuk-frontend/govuk/all";
-@import "lib/hmrc-frontend/hmrc/all";
+$govuk-include-default-font-face: false;
+@import "lib/govuk-frontend/govuk/base";
 
 @import "partials/shame";
 
 .summary-title {
-  @extend .govuk-heading-m;
+  @extend .govuk-heading-m !optional;
```

Comment:
Don't know if this should be optional, or keep the relevant imports

Reply:
Keeping it as it was stops the code from compiling.

Reply:
I think this one will fix it, 
`@import "lib/govuk-frontend/govuk/all";`

Reply:
That is one of the one we are meant to replace from the ticket description.

Reply:
Yeah, may be worth confirming with Jack. I read it that we should remove  - but double check there are no errors

Reply:
Ah interesting. Yes suspect we can keep the `@import "lib/govuk-frontend/govuk/all";` in that case